### DISCUSSION TITLE
what does the "fact" mean?

### DISCUSSION THREAD

**Discussion by lingsamuel** [WittleWolfie/OwlcatModdingWiki#1] [General]
I'm confused about this word. Does it mean an entity "factor"? It appears everywhere, entity fact, unit fact, blueprint fact, etc.
What's the difference between facts and parts in EntityDatabase? Both seem to be "component" of the ECS.

**Comment by WittleWolfie** [comment]
I'd ask in the discord: https://discord.gg/ut8zPzrC (mod-dev-technical).

I haven't delved too deep into this in particular, but most usage of "Fact" I've seen defines a mechanical property of a unit.

E.g. BlueprintUnitFacts can represent character abilities, buffs, or the characteristics of a unit, as seen in BlueprintUnit.

I haven't worked at all w/ parts so I'm not sure the distinction.

**Reply by lingsamuel** [reply]
Thank you for your reply. This is the first time I know there is an official modding channel...
